contracts/utility/Upgradeable.sol

```diff
@@ -83,6 +83,9 @@ abstract contract Upgradeable is IUpgradeable, AccessControlEnumerableUpgradeabl
      */
     function postUpgrade(bytes calldata data) external {
         uint16 initializations = _initializations + 1;
+        if (initializations < version()) {
```

Comment:
You can cache `version()`, but it's not the end of the world

Reply:
fixed